templates/rhdm70-kieserver-s2i.yaml

```diff
@@ -11,179 +11,179 @@ metadata:
-  description: 'Custom hostname for https service route.  Leave blank for default
-    hostname, e.g.: secure-<application-name>-execserv-<project>.<default-domain-suffix>'
+  description: 'Custom hostname for https service route.  Leave blank for default hostname, 
+    example: secure-<application-name>-execserv-<project>.<default-domain-suffix>'
   name: EXECUTION_SERVER_HOSTNAME_HTTPS
   value: ''
   required: false
 - displayName: Server Keystore Secret Name
-  description: The name of the secret containing the keystore file
+  description: The name of the secret containing the keystore file for Decision Server.
   name: HTTPS_SECRET
   value: kieserver-app-secret
   required: false
 - displayName: Server Keystore Filename
-  description: The name of the keystore file within the secret
+  description: The name of the keystore file within the secret.
   name: HTTPS_KEYSTORE
   value: keystore.jks
   required: false
 - displayName: Server Certificate Name
-  description: The name associated with the server certificate
+  description: The name associated with the server certificate.
   name: HTTPS_NAME
   value: jboss
   required: false
 - displayName: Server Keystore Password
-  description: The password for the keystore and certificate
+  description: The password for the keystore and certificate.
   name: HTTPS_PASSWORD
   value: mykeystorepass
   required: false
 - displayName: KIE Server Container Deployment
-  description: 'KIE Server Container deployment configuration in format: containerId=groupId:artifactId:version|c2=g2:a2:v2'
+  description: 'KIE Server Container deployment configuration in format: containerId=groupId:artifactId:version|c2=g2:a2:v2. 
+    This information identifies the decision service (KJAR file) that is built from your source. You can provide two 
+    or more KJAR files using the `|` separator, for example: `containerId=groupId:artifactId:version|c2=g2:a2:v2`. 
+    The Maven build process must produce all these files from the source in the Git repository.'
   name: KIE_SERVER_CONTAINER_DEPLOYMENT
   value: rhdm-kieserver-hellorules=org.openshift.quickstarts:rhdm-kieserver-hellorules:1.4.0-SNAPSHOT
   required: false
 - displayName: Git Repository URL
-  description: Git source URI for application
+  description: The URI for the Git repository containing the source for your decision service.
   name: SOURCE_REPOSITORY_URL
   value: https://github.com/jboss-container-images/rhdm-7-openshift-image.git
   required: true
 - displayName: Git Reference
-  description: Git branch/tag reference
+  description: Git branch/tag reference for the source of your decision service.
   name: SOURCE_REPOSITORY_REF
   value: rhdm70-dev
   required: false
 - displayName: Context Directory
-  description: Path within Git project to build; empty for root project directory.
+  description: Path within the Git project where your source must be built; empty for root project directory.
```

Comment:
If I understand this description right, it says that this property set some build directory (probably in container).
This env. variable says where is the pom.xml file to build from copied git repository (env. variable SOURCE_REPOSITORY_URL)

Reply:
IMHO this description does not match the variable purpose, as @jakubschwan  said.
Would be better use something like: "Git module location, if empty the root uri will be used to build."